buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/BuildForkPointDistribution.java

```diff
@@ -137,9 +131,16 @@ private void tryBuildDistribution(String commit) throws Exception {
             "-PtoolingApiShadedJarInstallPath=" + getDestinationFile("gradle-tooling-api-" + baseVersion + "-commit-" + commit + ".jar")
         };
 
-        if (buildScanRequired) {
-            String[] buildScanParams = new String[]{"--init-script", new File(getGradleCloneTmpDir(), "gradle/init-scripts/build-scan.init.gradle.kts").getAbsolutePath()};
-            return Stream.of(commands, buildScanParams).flatMap(Stream::of).toArray();
+        if (System.getenv().containsKey("CI")) {
+            String[] ciParams = new String[]{
+                "--init-script",
+                new File(getGradleCloneTmpDir(), "gradle/init-scripts/build-scan.init.gradle.kts").getAbsolutePath(),
+                "--build-cache",
+                "-Dgradle.cache.remote.url=" + System.getProperty("gradle.cache.remote.url"),
```

Comment:
Shouldn't we better query `settings.remote.credentials` instead of using the system properties? That would also change the if: configure the build cache for the executed build in the same way as the build cache for the root build.

Reply:
Thanks! I wasn't aware of this. Now I read these values from `settings.remote.credentials`, can you take a look at it?